.github/workflows/jgiven_branch_and_pr_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Java CI with Gradle
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build JGiven on Java ${{ matrix.Java }} with Junit ${{matrix.junit}}
```

Comment:
Is it necessary to use `${{matrix.java}}` instead of `${{ matrix.*J*ava }}` here?

Reply:
github doesn't seem to care about capitalization. But you're right. I should be consistent about it